docs/TuxxAnimations.md

```diff
@@ -111,93 +118,138 @@ A numeric id can be specified, in which case all elements will receive that key.
 
 ```javascript
-  <Fly duration={2000} delay={1000}>
+  <Fade duration={2000} delay={1000}>
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
 ##### <a id="animationProps-easing"></a>Easing Property - type: STRING - optional [#](#animationProps-easing)
 The easing property determines what motion path the animation takes when presented. Easing only accepts a string which can take any custom cubic bezier animation path you wish. For example:
 
 ```javascript
-  <Fly duration={3000} delay={1000} easing="cubic-bezier(0.680, -0.550, 0.265, 1.550)">
+  <Fade duration={3000} delay={1000} easing="cubic-bezier(0.680, -0.550, 0.265, 1.550)">
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
 The Easing property also accepts any of the Tuxx default easings as a string:
 
 ```javascript
-  <Fly duration={3000} delay={1000} easing="easeInOutBack">
+  <Fade duration={3000} delay={1000} easing="easeInOutBack">
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
 The Tuxx default easings and their motion paths are illustrated below. The 'Linear' path is not shown, but it is purely a straight line motion path.
 
 ![Easings](http://i.imgur.com/2XAGQgq.png "Easings")
 
+##### <a id="animationProps-vector"></a>Vector Property - type: OBJECT - optional - requires `Origin` & `Magnitude` [#](#animationProps-vector)
+The vector property allows you set where your animation will orignate from and the magnitude of that entrance and exit to that origin. The vector property ccepts an object and expects two keys within that object: `origin` and `magnitude`. Magnitude accepts a number value and Origin accepts any single item of or any combination of two items from the the following list of acceptable origins:  'top', 'bottom', 'left' and 'right'. For example, if you wanted your elements to originate from the bottom right corner of the containing animation component with a magnitude of 50 pixels, it's as easy as:
+
+```javascript
+  <Fade duration={3000} delay={1000} easing="easeInOutBack" vector={{origin: 'bottom right', magnitude: 50}}>
+    <h1>Hello World</h1>
+  </Fade>
+```
+
+
 ##### <a id="animationProps-custom"></a>Custom Property - type: OBJECT - optional [#](#animationProps-custom)
-The custom property accepts an object as an input with any CSS properties you wish the animation to include in addition to the default CSS transitions inherent to the particular component. For Example, if you wanted the element you've wrapped with a Fly animation component to also animate to the color red and increase its font size to 14px, it's as easy as:
+The custom property accepts an object as an input with any CSS properties you wish the elements within the animation component to transition to on enter. For example, if you wanted the element you've wrapped with a Fade animation component to also animate to the color red and increase its font size to 50px, it's as easy as:
```

Comment:
Recommend changing the wording at the end of the last sentence away from "...it's as easy as:" as it is used in the previous paragraph of text.